src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -18,6 +18,6 @@
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.setAddressBook(new AddressBook());
-        return new CommandResult(MESSAGE_SUCCESS);
+        return new CommandResult(MESSAGE_SUCCESS, false, false, false, false);
```

Comment:
Should we extract these booleans to be more readable? Something like isPerson, isExit, isHelp etc.